locale/rma_ch_de/wlprinter

```diff
@@ -1,6 +1,7 @@
 $self{charset} = 'ISO-8859-1';
 
 $self{texts} = {
+  'No Entries'                  => 'Keine Einträge zur Auswahl',
```

Comment:
why is he ä here shown, while the others have unknown symbol? Has this file some different envcoidng?